cdk.py

```diff
@@ -1,12 +1,30 @@
 #!/usr/bin/env python3
 from aws_cdk import App
+import aws_cdk as cdk
+
+import subprocess
 
 from cdk import config, stack
 
 deployment = config.Deployment(_env_file=".env")
 
 app = App()
 
+git_sha = subprocess.check_output(["git", "rev-parse", "HEAD"]).decode().strip()
+try:
+    git_tag = subprocess.check_output(["git", "describe", "--tags"]).decode().strip()
+except subprocess.CalledProcessError:
+    git_tag = "no-tag"
+
+tags = {
+    "Project": "veda",
+    "Owner": deployment.owner,
+    "Client": "nasa-impact",
+    "Stack": deployment.stage,
+    "GitCommit": git_sha,
+    "GitTag": git_tag,
+}
+
```

Comment:
very nice

Reply:
Yes this is awesome!
I think we need the this dropped in the StacIngestionApi construct around line 32?